site/templates/tv/detail.jet

```diff
@@ -80,7 +80,7 @@
 
           {{if len(tvseason.Episodes) > 0}}
             <div class="meta-detail-episodes-content">
-              <h2>{{i18n("meta_detail_episodes_title")}}</h2>
+              <h2 class="h3-style">{{i18n("meta_detail_episodes_title")}}</h2>
```

Comment:
I'm being bit fussy but the` <h1>` with class name as h2` <h2>` with class name as h3 looks bit confusing

Reply:
Yeah, agree, this is a no from me on this.